src/app/lib/config/services/portuguese.js

```diff
@@ -177,7 +177,7 @@ export const service = {
     },
     footer: {
```

Comment:
`noBylinesPolicy` should be updated to this anchor in the trust project page: `https://www.bbc.com/portuguese/institutional-50054434#authorexpertise`
`publishingPrinciples` should be updated to the trust project page: `https://www.bbc.com/portuguese/institutional-50054434`

Reply:
I just realised that this was also missed when we reviewed the Zhongwen config updates. https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/zhongwen.js#L266-L267 Could you please update this as well in this PR, so we don't have to open a new PR? Thank you.

Reply:
These are updated now